examples/plugins/cmd-runner/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.8
```

Comment:
Need to use a different image than alpine.

Reply:
Thanks; I got mixed up on which image I needed to use.

Reply:
Uses debian:stretch-slim now